content/lessons/css-layouts/responsive-layouts.md

```diff
@@ -38,7 +38,7 @@ preReadQuiz:
       type: checkbox
 
     # Question 3
-    - description: Which of the following values are valid for setting the "justify-content" property?
+    - description: Which of the following values is not valid for setting the "justify-content" property?
```

Comment:
Should this be `are not` if it's multiple choice?

Reply:
Maybe I can make it a radio button